src/js/components/Pagination/stories/DataTable.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Box, Grommet, DataTable } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+const PaginatedDataTable = () => {
+  const [select, setSelect] = React.useState([]);
+  return (
+    <Grommet theme={grommet} full>
+      <Box pad="large">
+        <DataTable
+          columns={columns}
+          data={[...DATA]}
+          onSelect={setSelect}
+          select={select}
+          sortable
+          step={3}
+          paginate
+        />
+      </Box>
+    </Grommet>
+  );
+};
+
+export { PaginatedDataTable as DataTable };
```

Comment:
I think this story should be under DataTable instead of Pagination, since that's the outer context.

Reply:
That works. I'd assume the same for the List story as well then. I can move them both.

Reply:
For the "Grid" story -- it's really an example of using Pagination as a controlled component vs integrated into DataTable/List. Do you feel like Grid is an appropriate title or should it be "Controlled"?

Reply:
I think Grid is fine